app/helpers/application_helper.rb

```diff
@@ -71,6 +71,11 @@ def humanized_bool(bool_value)
     bool_value ? 'Yes' : 'No'
   end
 
+  def auto_delius_import_enabled?(prison)
+    Flipflop.auto_delius_import? ||
+      (ENV['AUTO_DELIUS_IMPORT'] || '').split(',').include?(prison)
```

Comment:
If we want to set this env var, it'll need defining in https://github.com/ministryofjustice/offender-management-allocation-manager/blob/master/deploy/staging/shared-environment.yaml (or the prod/preprod equivalent).

Reply:
Yes - this is designed as a slow-rollout to a few prisons to see how they cope